templates/lune/data.json

```diff
@@ -0,0 +1,44 @@
+{
+  "lune": {
+    "head-tag": {
+      "extensions": [
+        "amp-sidebar"
+      ],
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Lato:400,700,900"
+      ]
+    },
+    "navbar": {
+      "name": "AMP Lune template",
+      "sidebar-id": "header-sidebar",
+      "sidebaronly": true,
+      "href": "lune.amp.html",
+      "logo": {
+         "name": "LUNE",
+         "classes": "h3 xs-hide"
+      },
+      "fixeditems": [
+        {
+          "link": {
+            "url": "mailto:contact@lune.com",
+            "text": "contact@l-u-n-e.com"
+          }
+        }
+      ],
+      "nav": [
+        {
+          "link": {"url": "product-blanc.amp.html", "text": "<span>B</span><span>l</span><span>a</span><span>n</span><span>c</span>"}
```

Comment:
can you see if amp-fit-text helps, i am not a fan of this, but if this gets us a good effect, we can probably overlook